x-pack/plugin/src/yamlRestTest/java/org/elasticsearch/xpack/test/rest/XPackRestIT.java

```diff
@@ -43,7 +43,7 @@ public class XPackRestIT extends AbstractXPackRestTest {
         .setting("xpack.searchable.snapshot.shared_cache.region_size", "256KB")
         .user("x_pack_rest_user", "x-pack-test-password")
         .feature(FeatureFlag.TIME_SERIES_MODE)
-        .feature(FeatureFlag.LEARNING_TO_RANK)
+        .feature(FeatureFlag.LEARN_TO_RANK)
```

Comment:
again, seems unnecessary? Merge in main?

Reply:
Same as before - fixed in 07613ae8876d9dc76ee60faf270d7dd534e324e9